api/member.go

```diff
@@ -130,5 +130,9 @@ func (a API) addNewMember(w http.ResponseWriter, req *http.Request) {
 
 	a.db.AddUserToDefaultResources(newMember.Email)
 
+	//TODO: [ML] Does this make sense here?  Should this be sent at all?  Maybe it should be invoked
+	//in a paypal webhook when a new subscription is created.
+	//mail.SendTemplatedEmail("welcome.html.tmpl", newMember.Email, "Welcome to hackRVA", newMember)
+
```

Comment:
I think the webhook makes sense.
This `addNewMember` function was added because of the delay that paypal has when a member signs up.  They don't show us the member via the api for at least 3 hours.

Also, we don't always add people with the `api.AddNewMember` function.  Ideally they are added automatically via the payments package (when we retrieve info from paypal) which calls `db.AddMembers(members []Member)`.

Reply:
Created #94 to implement webhooks